userdocs/conf.py

```diff
@@ -16,7 +16,6 @@
 
 extensions = [
     'sphinx.ext.graphviz',
-    'sphinx_reredirects',
```

Comment:
We need redirects to support redirects on the website. If we need to support an environment that doesn't support redirects, we'll need separate `conf.py` or some other dynamic way to handle it.

Reply:
If the pdf tag is defined for sphinx the redirects will be exlcuded